app/scripts/controllers/metametrics.js

```diff
@@ -329,19 +329,21 @@ export default class MetaMetricsController {
   // It sets an uninstall URL ("Sorry to see you go!" page),
   // which is opened if a user uninstalls the extension.
   updateExtensionUninstallUrl(participateInMetaMetrics, metaMetricsId) {
-    // TODO: Change it to the right URL once it's available.
-
     const query = {};
     if (participateInMetaMetrics) {
       // We only want to track these things if a user opted into metrics.
-      query.id = metaMetricsId;
+      query.mmi = Buffer.from(metaMetricsId).toString('base64');
       query.env = this.environment;
       query.av = this.version;
     }
     const queryString = new URLSearchParams(query);
-    this.extension.runtime.setUninstallURL(
-      `${EXTENSION_UNINSTALL_URL}?${queryString}`,
-    );
+
+    // this.extension not currently defined in tests
+    if (this.extension && this.extension.runtime) {
```

Comment:
we'll need this check for as long as we're not defining an `extension` param when instantiating the `MetaMetricsController` in testing. [metametrics.test.js#L127](https://github.com/MetaMask/metamask-extension/blob/2eb0fe6978ddb8a5492425d3883a35b1d96bab6a/app/scripts/controllers/metametrics.test.js#L127
)